.github/workflows/ci-semantic-pr.yml

```diff
@@ -0,0 +1,30 @@
+name: CI (Semantic PR)
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR Title
+    runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            docs
+            feat
+            fix
+            chore
+          scopes: |
+            core
+            en
+            ja
+          requireScope: true
```

Comment:
IMHO, 4 types and 3 scopes are enough for documentation site:

**Types**

- `docs` for any kinds of changes on documentation (e.g. fixing typos, new translation)
  - PRs whose commit message has `docs` type are supposed to be attached `lang:x` labels
- `feat` for application-level features (e.g. new page, new UI, etc.)
- `fix` for  application-level fixes
- `chore` for anything else (e.g. updating `README`, Change CI, etc.)

**Scopes**

- `core` for application
  - `docs(core)` would not be used in real operation
- `en` for English contents
- `ja` for Japanese contents

The reason why I defined both scopes and labels for i18n is that we can filter PRs and issues with labels, which is hard to achieve with scopes in commit massage only.